packages/realm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "realm",
-  "version": "12.0.0-rc.0",
+  "version": "12.0.0-alpha.2",
```

Comment:
Should it be the alpha here?

Reply:
Yes...that's strange that it isn't.  Good find